x/qgb/types/genesis.go

```diff
@@ -1,15 +1,67 @@
 package types
 
-// DefaultIndex is the default capability global index
-const DefaultIndex uint64 = 1
+import (
+	"fmt"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// DefaultParamspace defines the default qgb module parameter subspace
+const (
+	DefaultParamspace = ModuleName
+)
+
+// ParamsStoreKeyDataCommitmentWindow
+var ParamsStoreKeyDataCommitmentWindow = []byte("DataCommitmentWindow")
 
 // DefaultGenesis returns the default Capability genesis state
 func DefaultGenesis() *GenesisState {
-	return &GenesisState{}
+	return &GenesisState{
+		Params: &Params{
+			DataCommitmentWindow: 400,
+		},
+	}
 }
 
 // Validate performs basic genesis state validation returning an error upon any
 // failure.
 func (gs GenesisState) Validate() error {
+	// this line is used by starport scaffolding # genesis/types/validate
+	if err := gs.Params.ValidateBasic(); err != nil {
+		return sdkerrors.Wrap(err, "params")
+	}
+	return nil
+}
+
+// ParamKeyTable for qgb module
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// ParamSetPairs implements the ParamSet interface and returns all the key/value pairs
+// pairs of auth module's parameters.
```

Comment:
remove duplicate word

```suggestion
// ParamSetPairs implements the ParamSet interface and returns all the key/value
// pairs of auth module's parameters.
```